widget/playground/src/containers/StyleLayout/StyleLayout.CustomColors.tsx

```diff
@@ -0,0 +1,194 @@
+import type { CustomColorsTypes } from './StyleLayout.types';
+import type { Mode } from '../../store/config';
+import type { WidgetColorsKeys } from '@rango-dev/widget-embedded';
+
+import {
+  ChevronDownIcon,
+  ChevronUpIcon,
+  Collapsible,
+  CustomColorsIcon,
+  Divider,
+  Typography,
+} from '@rango-dev/ui';
+import React, { useEffect, useState } from 'react';
+
+import { ColorItemPicker } from '../../components/ColorItemPicker';
+import { WIDGET_COLORS } from '../../constants';
+import { useConfigStore } from '../../store/config';
+import { getMainColor } from '../../utils/colors';
+
+import {
+  ColoredCircle,
+  ColorsContent,
+  CustomColorCollapsible,
+  CustomColors,
+  FieldTitle,
+  Row,
+} from './StyleLayout.styles';
+
+const Colors = (props: { mainColor?: string; secondColor?: string }) => {
+  return (
+    <ColorsContent>
+      <ColoredCircle style={{ backgroundColor: props.mainColor }} />
+      {props.secondColor !== undefined && (
+        <ColoredCircle
+          position="absolute"
+          style={{
+            backgroundColor: props.secondColor,
+            right: -10,
+          }}
+        />
+      )}
+    </ColorsContent>
+  );
+};
+
+export function CustomColorsSection(props: CustomColorsTypes) {
+  const { tab, selectedPresets } = props;
+  const [openCustomColors, toggleCustomColors] = useState<{
+    tab: Mode;
+    value: boolean;
+  }>({ tab, value: false });
+  const [openCustomColor, setOpenCustomColor] = useState<string | null>(null);
+  const onChangeColors = useConfigStore.use.onChangeColors();
+  const isAutoTab = tab === 'auto';
+
+  const { theme } = useConfigStore.use.config();
+
+  const isOpenCustomColors =
+    tab === openCustomColors.tab && openCustomColors.value;
+
+  const handleOpenCollapse = (key: string) => {
+    if (openCustomColor === key) {
+      setOpenCustomColor(null);
+    } else {
+      setOpenCustomColor(key);
+    }
+  };
+  useEffect(() => {
+    setOpenCustomColor(null);
+  }, [tab]);
+
```

Comment:
For some themes, some colors are not applying, I don't know why!

![Recording 2023-10-30 at 16 20 26](https://github.com/rango-exchange/rango-client/assets/99180934/a2c2e7d4-e240-407c-b226-f63319f5c04e)

Reply:
On the home page, primary color is used for the button in enable mode
The color of secondary is also used to best root